packages/storybook/stories/CallComposite/Quickstart1ToNDocs.stories.mdx

```diff
@@ -32,7 +32,7 @@ Download the [UI Library quickstart composites](https://github.com/Azure-Samples
 ## Prerequisites
 
 - An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
-- [Node.js](https://nodejs.org/) Active LTS and Maintenance LTS versions (Node 14.19.0 and above).
+- [Node.js](https://nodejs.org/) Active LTS and Maintenance LTS versions (Node 14 latest).
```

Comment:
These docs refer to using the npm package, as opposed to using the repo directly which has rush, so _should_ work with any version 14+ -- if not we should definitely fix that

Reply:
Maybe I was too enthusiastic with find and replace...

The issue I have is that rush only works with 14 currently, which is a little behind the current LTS